tests/test_assets.py

```diff
@@ -450,16 +457,41 @@ def test_lookup_symbol(self):
                 self.assertEqual(result.symbol, 'EXISTING')
                 self.assertEqual(result.sid, i)
 
-    @parameterized.expand(
-        build_lookup_generic_cases()
-    )
-    def test_lookup_generic(self, finder, symbols, reference_date, expected):
+    def test_lookup_symbol_from_multiple_valid(self):
```

Comment:
@ssanderson Echoing Maya, yep, the case of overlapping symbols is fairly common.